composer.json

```diff
@@ -9,14 +9,14 @@
     "homepage": "https://laminas.dev",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0"
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0"
     },
     "require-dev": {
         "ext-phar": "*",
         "doctrine/annotations": "^2.0.1",
         "laminas/laminas-coding-standard": "^2.5.0",
-        "laminas/laminas-stdlib": "^3.17.0",
-        "phpunit/phpunit": "^10.3.3",
+        "laminas/laminas-stdlib": "^3.18.0",
+        "phpunit/phpunit": "^10.4.2",
```

Comment:
update to latest phpunit `^10.5.37` to fix lowest issue on php 8.4

```php
        "phpunit/phpunit": "^10.5.37",
```

then, regenerate composer.lock